velodyne_laserscan/src/velodyne_laserscan.cpp

```diff
@@ -183,25 +183,25 @@ void VelodyneLaserScan::recvCallback(const sensor_msgs::msg::PointCloud2::Shared
   // Construct LaserScan message
   if ((offset_x >= 0) && (offset_y >= 0) && (offset_r >= 0))
     {
-      const float RESOLUTION = std::abs(resolution_);
-      const size_t SIZE = 2.0 * M_PI / RESOLUTION;
+      const float resolution = std::abs(resolution_);
```

Comment:
Was there a reason for doing this? The "standard" is here from the Google C++ Style Guide: https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
So, they aren't macros, so they definitely shouldn't be all upper-case.  I could be convinced to use the `kResolution` form, but even then they aren't really constant; the resolution can be changed via parameter, so these are still variables in my mind.

That being said, I won't insist on it, so if you want to go back to the old form I can do this.

Reply:
I would prefer the `kResolution` form just to denote that they are constant within the context of their scope, if you wouldn't mind.

Reply:
Done in 8ae7f648d91a27c4599c99058237fe7ca4e44dbf